### PR TITLE
Requires libarmcl for arm architecture

### PR DESCRIPTION
**Pull request by chunseoklee**
- Add libarmcl runtime dep into spec file

Change-Id: If9beb0722d2c4fa55ea811a6a5bd4053bd67c2e0
Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

**Comment by chunseoklee** [comment]
this will be cherrypicked into tizen 6.5 branch.

**Comment by chunseoklee** [comment]
no way to test this PR :)

**Review by hseok-oh — APPROVED** [review]

File: packaging/nnfw.spec [hunk 1]
```diff
@@ -39,6 +39,10 @@ BuildRequires:  python
 BuildRequires:  libarmcl-devel >= v20.05
 %endif
 
+%ifarch %{arm} aarch64
+Requires: libarmcl
+%endif
```
Comment (hseok-oh) [inline packaging/nnfw.spec:44]: We can merge this `ifarch-endif` with above `ifarch-endif`.

**Comment by chunseoklee** [comment]
We have a memory of removing this :) [link](https://github.sec.samsung.net/STAR/nnfw/pull/9479)

**Comment by chunseoklee** [comment]
It turns out that this PR is not required for image delivery.

**Pull request closed without merging**